test/csuite/wt4333_handle_locks/main.c

```diff
@@ -106,9 +106,14 @@ op(WT_SESSION *session, WT_RAND_STATE *rnd, WT_CURSOR **cpp)
 
     /* Loop to open an object handle. */
     for (i = __wt_random(rnd) % uris; !done; __wt_yield()) {
-        /* Use a checkpoint handle for 50% of reads. */
-        ret = session->open_cursor(session, uri_list[i], NULL,
-          readonly && (i % 2 == 0) ? "checkpoint=WiredTigerCheckpoint" : NULL, &cursor);
+        /*
+         * Use a checkpoint handle for 50% of reads.
+         *
+         * FIXME: Checkpoint cursors are known to have issues in durable history so we've removing
+         * the use of checkpoint handles in this test. Depending on how we end up resolving this, we
+         * should either re-enable the testing of checkpoint cursors or remove this comment.
```

Comment:
I'd rather track the possibility of adding checkpoint cursors here in a JIRA ticket, instead of an embedded comment. i.e: could you remove the comment and open a JIRA ticket.

Reply:
8d7a007

I've referenced the ticket number in the comment now. I've left the comment there since I think it's easier if the person knows where to look and what lines to `git blame`.

If you feel strongly that it should be removed entirely, I can do that.

Reply:
OK - I'll stop pushing.